kentikapi/client.go

```diff
@@ -31,6 +38,13 @@ type Client struct {
 	Alerting *resources.AlertingAPI
 
 	config Config
+
+	// cloudexport
+	CloudExportAdminServiceAPI *cloudexport.CloudExportAdminServiceApiService
```

Comment:
I think that API object fields should be located next to each other to increase cohesion. Please move config at the bottom. The same with Client initialization (in NewClient)

Reply:
Done